README.md

```diff
@@ -569,26 +631,64 @@ To check the status of the website, you can access this URL:
 
 ### SilDisco module for SAML Discovery
 
+A SimpleSAMLphp module containing a custom IdP Discovery class and four Authentication Processing
+filters. It is meant to be used as a SAML Hub, also known as a SAML Proxy. For more information, see
+the [Module Overview](./docs/overview.md) in the docs/ folder.
+
 #### Configuration
 
-Ensure the DYNAMO_* environment variables are set as shown in the local.env.dist file.
```

Comment:
So, DYNAMO variables aren't used any more?

Reply:
They are, but they're described in the other doc. Wanted to keep this brief here.